ansible_collections/arista/avd/roles/eos_designs/python_modules/l3_edge/utils.py

```diff
@@ -299,9 +299,9 @@ def _get_port_channel_member_cfg(self, p2p_link: dict, interface_name: str) -> d
         """
         peer = p2p_link["data"]["peer"]
         peer_interface = p2p_link["data"]["peer_interface"]
-        default_description = f"P2P_LINK_TO_{peer}_{peer_interface}"
+        default_description = f"P2P_LINK_TO_{peer}_{member['peer_interface']}"
         return {
-            "name": interface_name,
+            "name": member["interface"],
             "type": "port-channel-member",
             "peer": peer,
             "peer_interface": peer_interface,
```

Comment:
This peer_interface should also be set to member["peer_interface"]

Reply:
fixed